narwhal/types/src/primary.rs

```diff
@@ -767,7 +765,7 @@ pub type BlockResult<T> = Result<T, BlockError>;
 
 #[derive(Debug, Clone, Eq, PartialEq)]
 pub struct BlockError {
-    pub id: CertificateDigest,
+    pub digest: CertificateDigest,
```

Comment:
Ah well I've seen in the codebase using the terms id/digest for the same entities. I am not sure I would go with `digest` here. The fact that the id is of type "digest" is another matter.